### DISCUSSION TITLE
Use event attributes as parameters in actions

### DISCUSSION THREAD

**Discussion by caiquedestro** [cloud-custodian/cloud-custodian#7797] [AWS] [answered]
Hey everyone,

While trying to set a policy action to send a notification to Slack, I found what I thought to be a limitation. Maybe someone can help me find a different solution.

The destination of a Slack notification can be defined as shown on [this documentation](https://cloudcustodian.io/docs/tools/c7n-mailer.html). I was wondering if I can use the **User name** event attribute in one of the destinations, since, when using the AWS SSO in my organization, the user name shown is the person's email.

I couldn't find a way to do that anywhere on the documentation. Is that possible or am I just trying to do something stupid?

**Comment by ajkerrigan** [comment, marked as answer]
Hi there! There are a couple special destination values mentioned [in the email section](https://cloudcustodian.io/docs/tools/c7n-mailer.html#configuring-a-policy-to-send-email) of the c7n-mailer docs, but which should apply to Slack delivery as well. For your case, using `event-owner` as a recipient is worth a shot.

For the record, you're not doing anything stupid :). Pinging the person who triggered an event makes a lot of sense!
